docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Serial Vault User Guide
+table_of_contents: true
+---
+
+# Serial Vault user guide
+
+The Serial Vault is a Canonical-hosted web service that is required to use a
```

Comment:
The wording is a little funny, I misread it the first time. Maybe instead say "The Serial Vault is a Canonical-hosted web service that is required to have a Secure Brand Store."